### DISCUSSION TITLE
feat(ListView): BodyTemplate 添加一个自定义css渲染不同风格例如间距

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#3414] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/8428709/0e030cdb-bca8-4e08-a381-625330127dcd)

**Comment by bb-auto** [comment]
@densen2014 Thank you for contacting us. We will give feedback later.

**Comment by densen2014** [comment]
@ArgoZhang 其实加不加都可以, 在父类控制也行. 老张你看着办事

**Reply by ArgoZhang** [reply]
这里是 `BodyTemplate` 的容器样式，按照设计是没有必要增加自定义样式的，如果有特殊需求，可以在 `BodyTemlate` 内自行增加样式，如果统一设置，可以在最外面的样式中设置，这里完全不需要，而且也没有合适的参数设置到这里。

还有就是组件内置了一些样式变量

```
--bb-lv-header-padding: #{$bb-lv-header-padding};
--bb-lv-header-bg: #{$bb-lv-header-bg};
--bb-lv-border-color: #{$bb-lv-border-color};
--bb-lv-item-trans: #{$bb-lv-item-trans};
--bb-lv-item-border-hover-color: #{$bb-lv-item-border-hover-color};
--bb-lv-item-shadow: #{$bb-lv-item-shadow};
--bb-lv-body-padding: #{$bb-lv-body-padding};
--bb-lv-body-item-margin: #{$bb-lv-body-item-margin};
--bb-lv-footer-padding: #{$bb-lv-footer-padding};
```

边距可以通过 `--bb-lv-body-item-margin` 自定义
